spec.html

```diff
@@ -32156,6 +32156,33 @@ <h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-time-zone-identifiers">
+        <h1>Time Zone Identifiers</h1>
+
+        <p>
+          Time zones in ECMAScript are described using <dfn variants="time zone identifier">time zone identifiers</dfn>, which must be Strings composed entirely of characters from the Unicode Basic Latin block.
+          Time zone identifiers are case-insensitive and so are compared using ASCII case-insensitive comparisons.
```

Comment:
```suggestion
          Time zone identifiers are compared using ASCII case-insensitive comparisons.
```

I find the "are case-insensitive" to be redundant. I also don't think we need to explain the rationale here anyway.